src/main/java/com/google/sps/servlets/RetrieveBasedOnId.java

```diff
@@ -68,14 +68,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // get all parameter names and its values from HTTP request
     final String question = "question";
     final String option = "choice";
+    final String roomID = "roomID";
+    final String IP = "IP";
```

Comment:
lets rename this to ipAddress to be consistent in coding style.

Reply:
Also here IP seems to be hard coded, right? Please update, also apply sha256 before storing in dp.